force-app/main/default/classes/Homework.cls

```diff
@@ -1,60 +1,79 @@
 public with sharing class Homework {
     // Declare a variable to hold the postcode of Ginetta House - as
     // the office cannot move, this should be immutable
+    static final String postCode = 'WS13 8XG';
 
 
     // Declare a string that is initialised from an Integer
-
+    Integer numberOfStudents = 20;
+    String stringValueOfnumberOfStudents = String.valueOf(numberOfStudents);
 
     public void newYearsEve()
     {
         // Create a variable representing the time 12:30:25 in the local 
         // timezone and use this time to create a DateTime for the date 
         // 12/31/2021
+
+        Time localTime = Time.newInstance(12, 30, 25, 0);
+        // Datetime myNewYearEveDateTime = Date.NewInstance(2021, 12, 31) + Time.newInstance(12, 30, 25, 0);
```

Comment:
Have a look at the DateTime class in the System namespace docs of the Apex developer's guide - there is a constructor that takes a Date and a Time parameter.